src/hooks/useLocale.jsx

```diff
@@ -0,0 +1,23 @@
+// import { useState, useEffect } from "react";
+import Spanish from "../lang/es-US.json";
+import English from "../lang/en-US.json";
+
+// detect the language preference of the user's browser
+const useLocale = () => {
+  const locale = window.navigator.userLanguage || window.navigator.language;
```

Comment:
💡 I think we only need `const locale = navigator.language;`.

Reply:
Addressed:  6447f3c407ffb09cf1bec564ec331c363a951e9f